apps/src/aiDifferentiation/AiDiffChat.tsx

```diff
@@ -102,13 +108,15 @@ const AiDiffChat: React.FC<AiDiffChatProps> = ({
 
   const [suggestionPage, setSuggestionPage] = useState(0);
 
+  const isCSP = curriculumCourses.includes('csp');
```

Comment:
This will be false while the curriculumCourses is loading. Is that ok or do we need a spinner or loading state? I assume it's fine since we are just adding more prompts to the end, but wanted to check.